examples/laptop.yml

```diff
@@ -42,8 +44,10 @@ config: examples/config.yml
 
 inputs:
     # See README for paths to download these files
-    shear_catalog: data/example/inputs/shear_catalog.hdf5
-    photometry_catalog: data/example/inputs/photometry_catalog.hdf5
+    shear_catalog:  /global/cscratch1/sd/jsanch87/TXPipe_data/shear_catalog_small.h5
```

Comment:
looks like these are still changed.

Reply:
right, so the photometry catalog now needs a true redshift, and the shear catalog needs the metacal version of the redshfits... I think the right thing to do is to change the example input files. how do I change that?